assignments/prototypes.js

```diff
@@ -23,6 +33,22 @@
   * should inherit destroy() from GameObject's prototype
 */
 
+  function CharacterStats(charecterAttrs){
+    GameObject.call(this, charecterAttrs);
+    this.thisthis = charecterAttrs.thisthis;
+  }
+  CharacterStats.prototype = Object.create(GameObject.prototype);
+
+  CharacterStats.prototype.checkifthisthis = function() {
```

Comment:
Ahh the naming. Try to avoid naming variables like `thisthis` or `that` because it is pretty confusing.